### PR TITLE
Completed The Task of Translation into Bengali

### PR DESCRIPTION
**Pull request by ahammadshawki8**
# What I have done:

- [x]  Created README-BANGLA.md file
- [x]  Added the README-BANGLE.md file's link in all other READMEs.
- [x]  Asked a member of my community to review the README-BANGLA.md file

## Comment:
I hope that these changes will be reviewed and the pull request will be accepted soon. Please let me know your suggestions/feedback.

### CODE REVIEW

**Comment by erictleung** [comment]
@ahammadshawki8 thanks for the translation! One thing I would suggest is to change your `README-BANGLA.md` file name to `README-BN.md` to be consistent with the [ISO 639-1](https://en.wikipedia.org/wiki/List_of_ISO_639-1_codes) language codes as the rest of the repository adheres to.

I hope that makes sense. Let me know if you have any questions.

**Comment by ahammadshawki8** [comment]
> @ahammadshawki8 thanks for the translation! One thing I would suggest is to change your `README-BANGLA.md` filename to `README-BN.md` to be consistent with the [ISO 639-1](https://en.wikipedia.org/wiki/List_of_ISO_639-1_codes) language codes as the rest of the repository adheres to.
> 
> I hope that makes sense. Let me know if you have any questions.

Thanks for your suggestion. I will change the filename and all the links as soon as possible.

**Comment by ahammadshawki8** [comment]
I have checked the README-BN.md file again. There is no bug remaining. So, I think it is ready for merging.
Please let me know if you have any suggestions.
🙏🏼

File: README-BG.md [hunk 1]
```diff
@@ -1,11 +1,12 @@
-<!-- Do not translate this -->
+<!-- Do not translate this! -->
```
Comment (erictleung) [inline README-BG.md:1]: Why was this changed?

```suggestion
<!-- Do not translate this -->
```

File: README-BG.md [hunk 2]
```diff
@@ -20,7 +21,7 @@
 </ul>
 </details>
 
- <!-- Do not translate this --> <!-- Този превод се стреми да е по-скоро смислен, отколкото прецизно точен -->
+ <!-- Do not translate this! --> <!-- Този превод се стреми да е по-скоро смислен, отколкото прецизно точен -->
```
Comment (erictleung) [inline README-BG.md:24]: And this?

```suggestion
 <!-- Do not translate this --> <!-- Този превод се стреми да е по-скоро смислен, отколкото прецизно точен -->
```

File: README-CN.md [hunk 3]
```diff
@@ -2,6 +2,7 @@
     <tr>
         <!-- Do not translate this table -->
         <td><a href="./README.md"> English </a></td>
+        <td><a href="./README-BN.md">বাংলা</a></td>
```
Comment (erictleung) [inline README-CN.md:5]: For consistency

```suggestion
        <td><a href="./README-BN.md"> বাংলা </a></td>
```

**Review by erictleung — CHANGES_REQUESTED** [review]
Just some small comments.

Otherwise, thanks for translating! I don't have knowledge of Bengali so I'm unable to appropriately review this translation. I will keep this open for someone else who knows Bengali to review these changes.

**Comment by ahammadshawki8** [comment]
Ok I will work on it and ask my friend to review who know Bengali very well.

Thanks.

**Review by EssamSami5155 — APPROVED** [review]
Everything looks fine in Bengali. There is no problem with the translation. I think it is ready for merging. Good job @ahammadshawki8 .

**Review by erictleung — APPROVED** [review]
LGTM! 👍 Thanks for taking the time to translate and update all the other translation files! ✨

**Comment by erictleung** [comment]
@EssamSami5155 thanks for your review!

**Comment by ahammadshawki8** [comment]
Thank you very much.

**Comment by AniketSawale** [comment]
I reviewed and it seems everything is fine in Bengali.

**Review by AniketSawale — COMMENTED** [review]
Everything is fine in the Bengali version.

**Comment by BKD471** [comment]
Yes everything seems fine with Bengali translation ..good job mate :)

**Pull request merged**